examples/example-gateway/build/clients/baz/baz_handler.go

```diff
@@ -0,0 +1,87 @@
+// Code generated by zanzibar
```

Comment:
This entire file is too much boilerplate to save a few lines. 

Can we just make the tests more boilerplate and delete this ?